release-notes/opensearch-release-notes-1.3.19.md

```diff
@@ -0,0 +1,39 @@
+# OpenSearch and OpenSearch Dashboards 1.3.19 Release Notes
+
+## Release Details
+
+[OpenSearch and OpenSearch Dashboards 1.3.19](https://opensearch.org/versions/opensearch-1-3-19.html) includes the following maintenance updates.
```

Comment:
There are not just maintenance updates according to what we have here. 
'includes the following bug fixes, infrastructure and maintenance updates'